### ISSUE TITLE
Writing tensor Functions to a VTK file

### ISSUE THREAD

**Issue by ctjacobs** [firedrakeproject/firedrake#538]
Please could support for writing tensor-valued Functions to a VTK file be added. The following code

```
from firedrake import *

mesh = UnitSquareMesh(10, 10)
fs = TensorFunctionSpace(mesh, "DG", 2)
f = Function(fs).interpolate(Expression([("1.0", "2.0"), ("3.0", "4.0")]))

File("stress.pvd") << f
```

fails with

```
Traceback (most recent call last):
  File "/fs4/w05/w05/sw05/test/stress_vtk.py", line 7, in <module>
    File("stress.pvd") << f
  File "/work/w05/w05/sw05/build/firedrake/firedrake/io.py", line 119, in __lshift__
    self._file << data
  File "/work/w05/w05/sw05/build/firedrake/firedrake/io.py", line 533, in __lshift__
    self._update_PVD(data)
  File "/work/w05/w05/sw05/build/firedrake/firedrake/io.py", line 551, in _update_PVD
    new_vtk << function
  File "/work/w05/w05/sw05/build/firedrake/firedrake/io.py", line 193, in __lshift__
    raise RuntimeError("Can't output tensor valued functions")
RuntimeError: Can't output tensor valued functions
```

**Comment by wence-** [comment]
This needs some thought.  `evtk`, which we use under the hood, doesn't writing tensor-valued spaces (although paraview does allow them).  There's a question here about how much effort we want to invest in VTU output because there are still ongoing plans to migrate to hdf5+XDMF, at which point I would advocate dropping vtu completely.  In the mean time, it doesn't look too difficult to update evtk to support writing tensor-valued fields, but I don't know what the paraview format is for tensors.

**Comment by wence-** [comment]
Fixed in #737.
